src/rTerminal.ts

```diff
@@ -304,3 +304,64 @@ export async function sendRangeToRepl(rng: vscode.Range): Promise<void> {
     await vscode.commands.executeCommand('editor.debug.action.selectionToRepl');
     editor.selections = sel0;
 }
+
+export class TerminalLinkProvider implements vscode.TerminalLinkProvider {
+    provideTerminalLinks(context: vscode.TerminalLinkContext, token: vscode.CancellationToken): vscode.ProviderResult<vscode.TerminalLink[]> {
+        const matches: vscode.TerminalLink[] = [];
+        const regex = /[-_a-zA-Z]+\.r(:[0-9]+)?(:[0-9]+)?/gi;
```

Comment:
Is `/[-_a-zA-Z]+\.[Rr](:[0-9]+)?(:[0-9]+)?/g` clear enough so that there's no need to use ignore-case here?

Reply:
Also, what about the case where the file name contains spaces?

Reply:
Thank you for the review! The regex change works nicely, thank you.

The space one is tricky I think. I had the link handler handle searching for files so that the link provider was more performant, but that does mean its hard to tell if a file actually exists...

If we did it purely to *testthat* specifications, we could check purely between brackets, e.g. the brackets in:

```
Error: (test function.r:2:5): functions work properly
```

Alternatively, we could have a list of the workspace files, and parse the text against the list?

Reply:
I wonder if this terminal link provider overrides the default handler where full paths like `/home/ken/test.R:10:5` could be already handled, i.e. if we only handle `(file.R:line:col)` case, will the default cases still work?

Reply:
>If we did it purely to testthat specifications, we could check purely between brackets, e.g. the brackets in:
I know the following packages that could produce file links on some condition or error:

* testthat
* lintr
* Rcpp
* cpp11
* tinytest

It seems only testthat uses `(file:line:col)` notation. If we limit the regex to only capture this, it would make the terminal link provider much less useful.